flexmeasures/data/tests/test_scheduling_sequential.py

```diff
@@ -135,6 +138,9 @@ def test_create_sequential_jobs(db, app, flex_description_sequential, smart_buil
     ), f"Battery cost should be -4.415 €, got {battery_costs} €"
     assert total_cost == -2.1775, f"Total cost should be -2.1775 €, got {total_cost} €"
 
+    # Check that the SOC data is saved
+    assert len(soc_sensors["Test Battery"].search_beliefs()) == 97
```

Comment:
Let's also check whether  the numbers and units match our expectations. Using `integrate_time_series`.